ducktape/command_line/defaults.py

```diff
@@ -15,7 +15,7 @@
 import os
 
 
-class ConsoleConfig(object):
+class ConsoleDefaults(object):
```

Comment:
I renamed since it's essentially internal, and I thought the name better reflected its purpose and the way it is actually used. Could roll back if necessary though